errdefs/http_helpers.go

```diff
@@ -61,8 +61,9 @@ func GetHTTPErrorStatusCode(err error) int {
 		}
 
 		logrus.WithFields(logrus.Fields{
-			"module":     "api",
-			"error_type": fmt.Sprintf("%T", err),
+			"module":        "api",
+			"error_type":    fmt.Sprintf("%T", err),
+			"error_message": err.Error(),
 		}).Debugf("FIXME: Got an API for which error does not match any expected type!!!: %+v", err)
```

Comment:
The error is already logged on this line.

Did you mean to update the code further below? https://github.com/moby/moby/blob/9370ea2624650ac12a0bc75c01036851de47dbea/errdefs/http_helpers.go#L103-L106

If so, it'd probably be better to use `WithError()` for this (while at it, perhaps also change `fmt.Sprintf("%d")` to `strconv.Itoa()`;

```go
logrus.WithError(err).WithFields(logrus.Fields{
	"module":      "api",
	"status_code": strconv.Itoa(statusCode),
}).Debug("FIXME: Got an status-code for which error does not match any expected type!!!")
```

Reply:
Yes `WithError` is a better way, I'll fix that. Also `logrus.Fields` is actually `map[string]interface{}`, I think just put `"status_code": statusCode` will be fine.

Reply:
> I think just put "status_code": statusCode will be fine.

Ah, yes, makes sense. I was thinking about that, but wasn't sure if it would print correctly (but didn't try 😅)